File: kungfu/source/algorithm/is-prime/boolean.php
<?php
function isPrime(int $number): bool
{
    if ($number <= 1) {
        return false;
    }
    for ($i = 2; $i <= $number - 1; ++$i)  {
        if ($number % $i == 0) {
            return false;
        }
    }
    return true;
}

// usage 
$result = isPrime(10); // false

File: kungfu/source/algorithm/is-prime/boolean.go
func isPrime(number int) bool {
	if (number <= 1) {
        return false;
	}
	for i := 2; i <= number - 1; i++ {
		if (number % i == 0) {
            return false;
        }
	}
    return true;
}

// usage
var result bool = isPrime(10); // false